.gitignore

```diff
@@ -4,4 +4,5 @@ dist
 coverage
 *.log
 .reports
-.env
\ No newline at end of file
+.env
+.idea
```

Comment:
This should be in your global gitignore :)

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c